<launch>
	<node name="ar_pose" pkg="ar_pose" type="ar_multi" respawn="false">
		<param name="marker_pattern_list" type="string" value="$(find visnav_project)/marker/two_marker"/>
		<param name="threshold" type="int" value="100"/>
 		<remap from="/usb_cam/image_raw" to="/ardrone/image_raw"/>
 		<remap from="/usb_cam/camera_info" to="/ardrone/camera_info"/>
	</node>
	<node name="joystick_driver" pkg="joy" type="joy_node">
	  <param name="dev" type="string" value="/dev/input/js0"/>
	</node>
	<node name="visnav_project" pkg="visnav_project" type="visnav_project" />
	<node name="ardrone_joystick" pkg="ardrone_joystick" type="ardrone_teleop" output="screen" /> 
	<node name="dynamic_reconfigure_gui" pkg="dynamic_reconfigure" type="reconfigure_gui" /> 
	

	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find visnav_project)/launch/rviz.vcg" />
</launch>
